### ISSUE TITLE
JavaScript version cannot translate 👨‍👩‍👧‍👦 correctly.

### ISSUE THREAD

**Issue by qzchenwl** [Ranks/emojione#364]
## Javascript Conversion

![family-js](https://cloud.githubusercontent.com/assets/146802/18942296/98130040-85dd-11e6-8a4b-69441edee18b.png)
## PHP Conversion

![family-php](https://cloud.githubusercontent.com/assets/146802/18942301/9a6b5770-85dd-11e6-9b49-a4d1a713cbfa.png)

**Comment by caseyahenson** [comment]
@qzchenwl will you paste the code point string that you input here?

**Comment by qzchenwl** [comment]
@caseyahenson `String.fromCharCode(55357, 56424, 8205, 55357, 56425, 8205, 55357, 56423, 8205, 55357, 56422)`

In the console of page http://git.emojione.com/demos/latest/jsunicodetoimage.html I run the following code:

```javascript
emojione.unicodeToImage(String.fromCharCode(55357, 56424, 8205, 55357, 56425, 8205, 55357, 56423, 8205, 55357, 56422))
"👨‍👩‍👧‍👦" // which should be a image tag.

// The correct behavior looks like this.
emojione.unicodeToImage('😄')
"<img class="emojione" alt="😄" src="./assets/svg/1f604.svg?v=2.2.5"/>"
```

**Comment by qzchenwl** [comment]
@caseyahenson I inspected the PHP version, and found "👨‍👩‍👧‍👦" is translated to `./assets/svg/1f468-1f469-1f467-1f466.svg?v=2.2.3`. So the code point must be `1f468-1f469-1f467-1f466`, which is "👨", "👩", "👧" and "👦".

**Comment by caseyahenson** [comment]
@qzchenwl This should be resolved as of the 2.2.7 release.
